src/alfasim_sdk/_internal/types.py

```diff
@@ -954,7 +954,7 @@ class AddPhase:
 @attr.s(kw_only=True, frozen=True)
 class UpdatePhase:
     """
-    Allows the plugin update existing phases of the Hydrodynamic model.
+    Allows the to plugin update existing phases of the Hydrodynamic model.
```

Comment:
Maybe this should be:

> It allows [...]

or 

> Allow [...]

Check how this is done along the docs to keep the convention used, but I think it could be strange to start a sentence like this.

Reply:
Agreed, done!